app/src/main/java/com/kamron/pogoiv/OCRHelper.java

```diff
@@ -195,7 +195,7 @@ private String getPokemonNameFromImg(Bitmap pokemonImage) {
      * @return the candy name, or "" if nothing was found
      */
     private String getCandyNameFromImg(Bitmap pokemonImage) {
-        Bitmap candy = Bitmap.createBitmap(pokemonImage, widthPixels / 2, (int) Math.round(heightPixels / 1.3724285), (int) Math.round(widthPixels / 2.057), (int) Math.round(heightPixels / 38.4));
+        Bitmap candy = Bitmap.createBitmap(pokemonImage, widthPixels / 2, (int) Math.round(heightPixels / 1.3724285), (int) Math.round(widthPixels / 2.1), (int) Math.round(heightPixels / 38.4));
```

Comment:
To clarify what I'm scared (a bit) of: we have magic numbers with 7 significant digits, yet they can be fixed this easily?